tests/rptest/services/redpanda.py

```diff
@@ -2254,7 +2254,7 @@ def remove_local_data(self, node):
 
     def redpanda_pid(self, node):
         try:
-            cmd = "ps ax | grep -i 'redpanda' | grep -v grep | grep -v 'version'| grep -v \"]\" | awk '{print $1}'"
+            cmd = "ps ax | grep -i 'redpanda' | grep -v grep | grep -v 'version'| grep -v \"\[redpanda\]\" | awk '{print $1}'"
```

Comment:
It's getting difficult to know what we're trying to achieve. Can we use `pidof redpanda`?

Reply:
Oh cool suggestion, I can try that out and see what happens!